RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py

```diff
@@ -62,3 +62,14 @@
     OrderedHitsFactoryPSet = dict(maxElement = 100000),
     RegionFactoryPSet = dict(RegionPSet = dict(ptMin = 0.3)),
 )
+eras.trackingPhase2PU140.toModify(photonConvTrajSeedFromSingleLeg,
+    ClusterCheckPSet = dict(
+        MaxNumberOfCosmicClusters = 1000000,
+        MaxNumberOfPixelClusters = 100000,
+        cut = None
+    ),
+    OrderedHitsFactoryPSet = dict(maxElement = 100000),
+    RegionFactoryPSet = dict(RegionPSet = dict(ptMin = 0.3)),
+    primaryVerticesTag   = cms.InputTag("offlinePrimaryVertices"),
```

Comment:
Since I was not sure about the differences and in the version imported from 62XSLHCDEV there was `offlinePrimaryVertices`, I decided to stay with that. Do you think should I change it?